test/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../person'
+require_relative '../book'
+require_relative '../rental'
+
+describe 'Create person object' do
+  context 'given the valid inputs' do
+    it 'should create a person with age = 10' do
+      person = Person.new(20)
+      expect(person.age).to eql(20)
+    end
+
+    it 'should create a person with age = 20 and called Omar' do
+      person = Person.new(20, 'Omar')
+      expect(person.age).to eql(20)
+      expect(person.name).to match('Omar')
+    end
+
+    it 'The can_use_services? method should return true' do
+      person = Person.new(20, 'Kerolous')
+      expect(person.can_use_services?).to be true
+    end
+  end
```

Comment:
- [ ]  Good job for the passing tests, I have checked on your tests and I found that you've just tests only two methods of your classes. You should write unit tests for all the methods of all your classes such: tests for 
- add_rentals 
- correct_name
- teach_stuff
etc.
This helps to detect and protect against future bugs in your application.